pkg/converter/converter_test.go

```diff
@@ -57,13 +57,17 @@ spec:
 `
 
 func TestConvertUnstructured(t *testing.T) {
```

Comment:
I think it would be useful to also try to do this for the bundle / component objects. Maybe better as a follow-up PR

Reply:
You can open an issue, and assign me on it. Will be glad to continue working.

Reply:
You can open issue and assign me on it, will be glad to continue working.